pybleau/app/ui/tests/test_filter_expression_editor.py

```diff
@@ -0,0 +1,87 @@
+from unittest import TestCase
+
+from app_common.apptools.testing_utils import assert_obj_gui_works
+from pandas import DataFrame
+from traits.testing.unittest_tools import UnittestTools
+
+from pybleau.app.ui.filter_expression_editor import \
+    FilterExpressionEditorView, COPY_TO_CLIPBOARD
+
+
+class TestFilterExpressionEditor(TestCase, UnittestTools):
+    def setUp(self):
+        self.df = DataFrame(
+            {
+                "a": [1, 2, 4, 5],
+                "b": list("zxyh"),
+                "c_column": list("test"),
+                "d": ["1 alpha", "MM_TT_BB_03-92", "2zeta", "spacE Bar"],
+                "e_column": ["03_06 04_STANDARD.name",
+                             "about this", "._ a93", "d!@#.$% ^&*()_+d"]
+            },
+            index=[0, 1, 3, 4]
+        )
+
+    def test_bring_up_plot_manager_view(self):
+        view = FilterExpressionEditorView(expr="a == 4",
+                                          source_df=self.df)
+        assert_obj_gui_works(view)
+
+    def test_click_type_changed(self):
+        view = FilterExpressionEditorView(expr="a == 4",
+                                          source_df=self.df)
+        self.assertTrue(view.auto_append)
+        with self.assertTraitChanges(view, "auto_append"):
+            view.click_type = COPY_TO_CLIPBOARD
+        self.assertFalse(view.auto_append)
+
+    def test_edit_iniaialized_True(self):
+        view = FilterExpressionEditorView(expr="a == 4",
+                                          source_df=self.df)
+        self.assertTrue(view._edit_initialized)
```

Comment:
If this needs testing, the attribute should be public. Otherwise, maybe you want to test something else, such as the fact that since `edit_initialized` is True, you can do something...

Reply:
Made it public. No real need for it to be private.